backend/classes/production/sefilm/application.properties

```diff
@@ -0,0 +1,16 @@
+#Database Configuration
```

Comment:
This file should not be committed. I thought I had removed the folder from .gitignore...